src/acquisition/covid_hosp/common/test_utils.py

```diff
@@ -57,3 +82,17 @@ def load_sample_metadata(self, metadata_name='metadata.csv'):
 
   def load_sample_dataset(self, dataset_name='dataset.csv'):
     return pandas.read_csv(self.data_dir / dataset_name, dtype=str)
+
+  def truncate_tables(self, database, tables):
+    # use the local instance of the Epidata API
+    Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
+
+    # use the local instance of the epidata database
+    secrets.db.host = 'delphi_database_epidata'
+    secrets.db.epi = ('user', 'pass')
+
+    # clear relevant tables
+    with database.connect() as db:
+      with db.new_cursor() as cur:
+        for table in tables:
+          cur.execute(f'truncate table {table}')
```

Comment:
if we cant do this reflectively (and have to specify each table instead), perhaps this can just be left as a TODO to be addressed after #1126